packages/dds/tree/src/core/rebase/changeRebaser.ts

```diff
@@ -103,21 +103,39 @@ export interface TaggedChange<TChangeset> {
 	 * Considered false if undefined.
 	 */
 	readonly isRollback?: boolean;
+	/**
+	 * When populated, indicates the original intention that the changeset is based on.
+	 * When omitted, the `tag` field represents the intention.
```

Comment:
Do you mean the `revision` field?

Reply:
I did, I see there's more of that in the paragraph.
What do you think of make the field names the same between `TaggedChange` and `RevisionInfo`?
Also, do you prefer `tag` or `revision`?

Reply:
I do think it's a bit odd that RevisionInfo uses the term `tag` while basically everywhere else uses `revision`. I'm not sure I have a preference between the names.